### ISSUE TITLE
Movie list search UI

### ISSUE THREAD

**Issue by adelikat** [adelikat/tasvideos#264]
Similar to what's on legacy site movie pages (collapsed by default), we should provide a UI to search movies by the various tokens that are available

**Comment by meshuggahtas** [comment]
https://tasvideos.org/Publications/Filter is this the page to be worked on?

**Comment by adelikat** [comment]
Yes, that page satisfies this ticket, I just forgot this ticket existed, haha
